core/debugger/remote_debugger_peer.h

```diff
@@ -43,12 +43,12 @@ class RemoteDebuggerPeer : public RefCounted {
 
 public:
 	virtual bool is_peer_connected() = 0;
+	virtual int get_max_message_size() const = 0;
```

Comment:
I just reordered the methods here to make sure both implementations follow the same order, so it's easy to see which methods are overridden or not.